### DISCUSSION TITLE
Destination Postgres: Normalization Error for Timestamp No Timezone Incorrect Type Cast

### DISCUSSION THREAD

**Discussion by corbin-Tprime** [airbytehq/airbyte#32256] [General]
Hello,

I am working with exporting data from `snowflake` to a `postgres` database. I am running into an error where our `dbt runs` are casting `timestamps without time zones` to `text` values. The error we are receiving is as follows:

```
HINT:  You will need to rewrite or cast the expression.
Database Error in model test_model_name_redacted(models/generated/airbyte_incremental/public/test_model_name_redacted.sql)
  column "created_at__dbt_alter" is of type timestamp without time zone but expression is of type text
  LINE 4: ..._model_name_stg" set "created_at__dbt_alter" = "created_a...

```

Snowflake Source Connector: `0.2.2`
Postgres Destination Connector: `0.4.0`
Airbyte Version:  `0.5.33`

**Comment by marcosmarxm** [comment]
You can try to export the dbt model, edit it and run as custom transformation.
Normalization module is deprecated and won't receive any updates. Airbyte team is working to upgrade destinations to use [Destination v2](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2) format but Postgres Destination wasn't worked yet.

**Reply by corbin-Tprime** [reply]
@marcosmarxm Could you please clarify what you mean by export the dbt model? Or how we could export the dbt model?

**Reply by marcosmarxm** [reply]
Here you can get more info about how extract the normalization project: https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-sql

**Reply by corbin-Tprime** [reply]
@marcosmarxm Does the sync job need to complete to access these files? I ran the steps in the documentation however, I got a could not find file error?

**Reply by marcosmarxm** [reply]
I don't think they need to complete but only need to start the Normalization step.

[a comment by an automated account is not shown]
